sdcm/logcollector.py

```diff
@@ -965,6 +967,25 @@ class SCTLogCollector(LogCollector):
     ]
     cluster_log_type = 'sct-runner'
     cluster_dir_prefix = 'sct-runner'
+    too_big_log_size = 3*1024*1024*1024
+
+    def _create_tar_file(self, src_path: str, src_name: str) -> list[str]:
```

Comment:
why not to name it `_compress_file`? - since it is not creating `tar` file always anymore :)

Reply:
I don't have strong opinions either way regarding this one, 
I don't think there a cost issue either way (seems like)

Reply:
My opinion is also not strong - I have optic fiber at my place :)

Reply:
If `2 different archives on S3` mean `2 separate archives on S3` then `yes`.
`sct.log` is always much bigger than all other `event` files and doesn't really need to be in a single archive with anything else.

So, I see gain in having 2 separate archives, but don't see any real harm in continue having all of it in a single archive.